instrumentation/kubernetes/kubernetes.gradle

```diff
@@ -0,0 +1,39 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+apply plugin: 'org.unbroken-dome.test-sets'
+
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+muzzle {
+  pass {
+    group = "io.kubernetes"
+    module = "client-java-api"
+    versions = "[7.0.0,)"
+    assertInverse = true
+  }
+}
+
+testSets {
+  latestDepTest {
+    dirName = 'test'
+  }
+}
+
+dependencies {
+  compileOnly(group: 'com.squareup.okhttp3', name: 'okhttp', version: '3.0.0')
```

Comment:
Does Kubernetes client depend on okhttp? This `compileOnly` here essentially assumes that whenever this instrumentation is applied, there will be okhttp present in that JVM as well. Is it true?

Reply:
removed okhttp dependencies, the kubernetes dependency already contains it